tzwhere/tzwhere.py

```diff
@@ -40,7 +49,7 @@ class tzwhere(object):
     DEFAULT_CSV = os.path.join(os.path.dirname(__file__),
                                'tz_world.csv')
 
-    def __init__(self, input_kind='json', path=None):
+    def __init__(self, input_kind='csv', path=None):
```

Comment:
How does the default `input_kind='csv'` work out of the box if only the gzipped csv is part of the package.